vicodyn/src/vicodyn/darkmetrics.cpp

```diff
@@ -0,0 +1,66 @@
+#include "cocaine/vicodyn/darkmetrics.hpp"
+
+#include <cocaine/context.hpp>
+#include <cocaine/dynamic.hpp>
+#include <cocaine/format/dynamic.hpp>
+#include <cocaine/format/exception.hpp>
+#include <cocaine/logging.hpp>
+#include <cocaine/unicorn/value.hpp>
+
+#include <blackhole/logger.hpp>
+
+namespace cocaine {
+namespace vicodyn {
+
+darkmetrics_t::darkmetrics_t(context_t& context, peers_t& peers, const dynamic_t& args)
```

Comment:
We have agreed to have a way to disable _system_weight_ mechanics within config? Is _disable switch_ implemented already or it would be in separate PR?

Reply:
Thank you, I'll add properties in config in current PR.

Reply:
Added to separate commit.